subsys/bluetooth/controller/Kconfig

```
@@ -199,7 +199,7 @@ config BT_CTLR_ISO_TX_BUFFERS
 	int "Number of Isochronous Tx buffers"
 	default 3
 	range 1 19
-	depends on BT_CTLR_ADV_ISO || BT_CTLR_SYNC_ISO || BT_CTLR_CENTRAL_ISO || BT_CTLR_PERIPHERAL_ISO
+	depends on BT_CTLR_ISO
```

Synchronized Receiver does not require Tx buffers. I have removed this in my draft PR: https://github.com/zephyrproject-rtos/zephyr/blob/3337aef1bcecb87fef7c0e829cd405b8e6807d12/subsys/bluetooth/controller/Kconfig#L209

```suggestion
	depends on BT_CTLR_ADV_ISO || BT_CTLR_CONN_ISO
```

---

Done. Added as separate commit to separate the pure renaming from the dependency fix.